### ISSUE TITLE
[Calico/Windows] How is SNAT configured on Windows when using VXLAN

### ISSUE THREAD

**Issue by eyenx** [projectcalico/calico#6324]
Hi, this is more a question as an issue.

We are having Problems with the SNAT on Windows Node using Calico. We see that Outgoing connection which are leaving the IPPool are taking somewhat the same SNAT Ports. We are presuming we run into some kind of SNAT Port exhaustion and our firewall is blocking traffic, cause a different connection is trying to use the same SNAT Port when connection to a different host (without sending any SYN Package).

We don't see this happening on our Linux nodes, and also normal curl testing shows us the Linux node take more randomly SNAT Ports.

We also tried configuring FelixConfiguration to use a specific NATPortRange, but as linux nodes where respecting that configuration, Windows nodes are still using a very high port range.

Can you help us understand in which context and where SNAT is configured on Windows nodes? 

We attached a Screenshot of the Firewall Logs as well as a log file of our curl/tcpdump & felixconfiguration.

![image001](https://user-images.githubusercontent.com/1947882/177613543-e9b65f5a-fd52-45c5-8728-1c9aecc2beca.png)
[curl_tcpdump.log](https://github.com/projectcalico/calico/files/9057076/curl_tcpdump.log)
[felixconfig.txt](https://github.com/projectcalico/calico/files/9057077/felixconfig.txt)

**Comment by sil-zh** [comment]
Hi, i have a update on this issue.

We tried to do some more debuggin on this. To see if kubernetes / calico respects the ports given from the windows operating system, we changed the "Default Dynamic Port Range for TCP/IP" on the Windows System to lower Ports (18000-18255)

We did this with the command:

`netsh int ipv4 set dynamicport tcp start=18000 num=255`

After this comment, we could see outgoing traffic from the Windows OS (for example Curl commands) used Source Ports 18000 - 18255. But commands from the pods running on this Windows Node (also for example curl) still used the high ports arround 65000

So - we think, that calico does not check / accept the ports given by the OS and not check / accept NATPortRange from CalicoFelix. So there is still the question, where this Source Ports are set for windows nodes running calico. Does anyone know?

any help or hint would be highly appreciated.
